sdk/computeschedule/arm-computeschedule/test/public/computeschedule_operations_test.spec.ts

```diff
@@ -34,7 +34,7 @@ describe("ComputeSchedule test", () => {
     await recorder.stop();
   });
 
-  it("operations list test", async () => {
+  it.skip("operations list test", async () => {
```

Comment:
@NneOkeke fyi, skip this test case as apiversion 2025-05-01 is not available

"ResponseBody": {
        "error": {
          "code": "InvalidResourceType",
          "message": "The resource type 'Operations' could not be found in the namespace 'Microsoft.ComputeSchedule' for api version '2025-05-01'. The supported api-versions are '2024-10-01,2024-08-15-preview,2024-06-01-preview'."
        }
      }